ut/print_ut.cpp

```diff
@@ -0,0 +1,412 @@
+    EXPECT_TRUE(correct_res == sstr.str());
+}
+
+TEST(PrettyPrintBlockTest, StringType) {
+    Block block;
+
+    auto str = std::make_shared<ColumnString>();
+    auto fixed_str = std::make_shared<ColumnFixedString>(10);
+
+    str->Append(
+        "Timeplus is a streaming-first data analytics platform. It provides powerful end-to-end capabilities to help teams process "
+        "streaming and historical data quickly and intuitively, accessible for organizations of all sizes and industries.");
+    str->Append("'./,[]{}()*&^%$#%$#@!?<>\":");
+
+    fixed_str->Append("stream sql");
+    fixed_str->Append("Timeplus");
+
+    block.AppendColumn("timeplus", str);
+    block.AppendColumn("fixd_string", fixed_str);
+
+    std::string correct_res = R"(
++--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+------------------+
+|                                                                                                                                                                                                                                   timeplus |      fixd_string |
++--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+------------------+
+|                                                                                                                                                                                                                                     string | fixed_string(10) |
++--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+------------------+
+| Timeplus is a streaming-first data analytics platform. It provides powerful end-to-end capabilities to help teams process streaming and historical data quickly and intuitively, accessible for organizations of all sizes and industries. |       stream sql |
+|                                                                                                                                                                                                                 './,[]{}()*&^%$#%$#@!?<>": |         Timeplus |
++--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+------------------+
+)";
+    std::stringstream sstr;
+    sstr << '\n' << PrettyPrintBlock{block};
+    EXPECT_TRUE(correct_res == sstr.str());
```

Comment:
can we use EXPECT_EQ macro? the difference is EXPECT_EQ will print both expected and actual value on assert failure so is easier for trouble shooting.